onnx/test/reference_evaluator_test.py

```diff
@@ -2611,6 +2611,104 @@ def test_mvn(self):
                 self.assertEqual(expected.shape, got.shape)
                 assert_allclose(expected, got)
 
+    def test_lp_pool_1d_1(self):
```

Comment:
Please take a look at https://github.com/onnx/onnx/pull/4761 ... is the test-coverage there adequate? In the long run, it would help to ensure that can serve as a complete test-suite for all onnx backends.

Reply:
I created the unit tests because there was no backend test to use. But I'll change that when PR #4761 is merged.